internal/pkg/cli/deploy/svc.go

```diff
@@ -98,20 +97,37 @@ type svcStackConfigurationOutput struct {
 	svcUpdater serviceForceUpdater
 }
 
-func validateAppVersionForAlias(appName string, appVersionGetter versionGetter) error {
+type errAppOutOfDate struct {
+	svc           string
+	curVersion    string
+	neededVersion string
+}
+
+func (e *errAppOutOfDate) Error() string {
+	return fmt.Sprintf("app version must be > %s", e.neededVersion)
```

Comment:
Should this be >= 🤔

Reply:
fixed in https://github.com/aws/copilot-cli/pull/4850!